README.md

```diff
@@ -1,11 +1,20 @@
 # interactive-examples
 
 [![Build Status](https://travis-ci.org/mdn/interactive-examples.svg?branch=master)](https://travis-ci.org/mdn/interactive-examples)
-
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 
 Home of the [MDN](https://developer.mozilla.org/) interactive code examples.
 
+## Project maintainers
+
+Should you have any questions regarding this project, please feel feel to @mention either @wbamberg or @schalkneethling
+
+## Good first issues
+
+Want to contribute to the interactive examples project? Here are a couple of good first issues to get you started. Thanks!
+
+[Good first issues](https://github.com/mdn/interactive-examples/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22example+needed%22+no%3Aassignee)
+
```

Comment:
I think this should be in CONTRIBUTING.md :)

Reply:
Yes and no. I wanted these to be visible when you just look at the repo without clicking through to anything else. We could have it there as well, and perhaps other links for folks who have contributed before. Thoughts?